db/db_utils.go

```diff
@@ -232,6 +222,7 @@ func RemoveAll(collection string, query any) error {
 }
 
 // Update updates one matching document in the collection.
+// DEPRECATED: Use UpdateContext instead.
```

Comment:
This is going to be removed once [DEVPROD-15398](https://jira.mongodb.org/browse/DEVPROD-15398) is completed? If so, can we link that ticket as a TODO so that the person implementing it knows to remove these functions when it's complete?

Same with the one below.

Reply:
Yeah sure, I'll add it to the code and link the lines in the ticket so they implementor has a general idea.